lib/routemap_cli.c

```diff
@@ -890,6 +890,76 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(set_min_metric, set_min_metric_cmd,
+	   "set min-metric <(0-4294967295)$metric>",
+	   SET_STR
+	   "Minimum metric value for destination routing protocol\n"
+	   "Minimum metric value\n")
+{
+	const char *xpath =
+		"./set-action[action='frr-route-map:set-min-metric']";
+	char xpath_value[XPATH_MAXLEN];
+	char value[64];
+
+	nb_cli_enqueue_change(vty, xpath, NB_OP_CREATE, NULL);
+
+	snprintf(xpath_value, sizeof(xpath_value),
+		 "%s/rmap-set-action/min-metric", xpath);
+	snprintf(value, sizeof(value), "%s", metric_str);
+
+	nb_cli_enqueue_change(vty, xpath_value, NB_OP_MODIFY, value);
+
+	return nb_cli_apply_changes(vty, NULL);
+}
+
+DEFPY_YANG(no_set_min_metric, no_set_min_metric_cmd,
+	   "no set min-metric [OPTVAL]",
```

Comment:
Does it mean CLI accepts even `no set min-metric ANY_STRING`?

Reply:
I modeled it after the `set metric` command, but I probably can just make simplify it here.